src/catalua.cpp

```diff
@@ -49,6 +49,9 @@ lua_State *lua_state = nullptr;
 // we know where to load files from.
 std::string lua_file_path = "";
 
+std::stringstream lua_output_stream;
+std::stringstream lua_error_stream;
```

Comment:
Mostly in-experience with the standard library. I was dealing with strings, I ```stringstream``` before, so I used it here.